app/konnect/runtime-manager/vaults/index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Secrets Management in Konnect
+content_type: explanation
+---
+
+Secure your {{site.konnect_short_name}} environment with centralized secrets,
+making it easier to manage and build governance.
+
+A secret is a sensitive piece of information required for proper gateway operations.
+Secrets may be part of the core gateway configuration,
+or part of gateway configuration associated with APIs serviced by the gateway.
+Some of the most common types of secrets include:
+* Certificates
+* API keys
+* Personal access tokens
+* Credentials for databases
+* Certain plugin fields, like `session_secret` in the OIDC plugin
+
+You can use vaults to safely store and retrieve secrets used in {{site.base_gateway}}
+deployments, improving the fundamental security of your applications.
+In configuration, you can then reference the secrets stored in vaults instead
+of displaying the actual value of the secret.
+
+This ensures that secrets are not stored in plaintext throughout the platform,
+in places such as `kong.conf` files, in declarative configuration files, logs,
+or in the {{site.konnect_short_name}} UI. This way, the {{site.konnect_short_name}}
+platform never stores or sees sensitive credentials, letting you use the security controls
+that you already have around sensitive information.
+
+## Vaults interface in {{site.konnect_short_name}}
+
+![Vaults interface](/assets/images/docs/konnect/konnect-vaults.png)
+> Figure 1: Overview page for all vaults configured for a runtime group.
+
+Number | Item | Description
+-------|------|------------
+1 | **Vaults menu link** | Main link to the vaults configuration for a runtime group. Appears when you select a runtime group.
+2 | **Add vault** | Click the **Add vault** button to set up any supported Konnect vault backend.
+3 | **Table entries for vaults** | Select a table entry to see the configuration for a particular vault. On each vault's configuration page, you can edit or delete the vault, or copy the entire configuration as JSON.
+4 | **Vault ID** | The vault's UUID. You can click the copy icon to copy the ID for use in configuration.
+5 | **Vault action menu** | Interact with a configured vault. From this menu, you can <b>view</b>, <b>edit</b>, or <b>delete</b> a vault's configuration. <br><br> <b>Be careful when deleting a configured vault!</b> Once deleted, a vault can't be restored, and all vault references will fail if not updated.
```

Comment:
```suggestion
5 | **Vault action menu** | From this menu, you can <b>view</b>, <b>edit</b>, or <b>delete</b> a vault's configuration. 
```

Reply:
I think the deleted part also applies to the information from within the configuration menu. If there is no UI text that appears that states that info when you try to delete a vault, then we should just add that information under the table in a note.

Reply:
There is UI text for this, so i'll just remove.